src/edu/harvard/CS50.java

```diff
@@ -256,9 +258,13 @@ public static long getLong(String prompt)
      */
     public static String getString(String prompt)
     {
+
+        // instantiate scanner once
+        if (s == null)
```

Comment:
`s` and all other methods are `static`, so `this` has no context in them. I think we may qualify with the class name instead (i.e., `CS50.s`), but that's implied anyway.

Can rename to `scanner` though.